generators/cypress/templates/src/test/javascript/cypress/integration/audits.spec.ts.ejs

```diff
@@ -0,0 +1,46 @@
+<%#
+ Copyright 2013-2021 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+describe('Audits', () => {
+    beforeEach(() => { 
+      cy.visit('/');
```

Comment:
It seems that we are trying to calculate the lighthouse score for the application landing page (`/` route). For more comprehensive coverage, we should do that for each route in our application.

Reply:
Do you expect huge differences between them? But of course that could be extended.

Reply:
TBH, I don't know. I haven't looked into cypress audits and it's capabilities. Do you have assertions to check on best practices? If yes, then, you may want to run the audit for each page.

In the `svelte` blueprint, I used a different approach and the integration is only in the CI (not in generated code). You can of course do that in the generated code, but that was not the concern that I want to handle at moment.